express-static/index.js

```diff
@@ -0,0 +1,16 @@
+var express = require('express');
+var path = require('path');
+var app = express();
+
+app.listen(3000, () => {
+  // eslint-disable-next-line no-console
+  console.log('Express server listening on port 3000');
+});
```

Comment:
Although this will work here, it is best to put the `listen` method as the last thing in the file.